core/templates/dev/head/components/share/SharingLinksDirective.js

```diff
@@ -29,24 +29,21 @@ oppia.directive('sharingLinks', [function() {
     },
     templateUrl: 'components/sharingLinks',
     controller: [
-      '$scope', '$window', '$injector', 'oppiaHtmlEscaper',
+      '$scope', '$window', 'oppiaHtmlEscaper', 'ExplorationEmbedButtonService',
       'siteAnalyticsService', 'UrlInterpolationService',
       function(
-          $scope, $window, $injector, oppiaHtmlEscaper,
+          $scope, $window, oppiaHtmlEscaper, ExplorationEmbedButtonService,
           siteAnalyticsService, UrlInterpolationService) {
         $scope.registerShareEvent = null;
 
         if ($scope.shareType === 'exploration') {
           $scope.explorationId = $scope.getExplorationId();
-          var ExplorationEmbedButtonService = $injector.get(
-            'ExplorationEmbedButtonService');
 
           $scope.activityType = 'explore';
           $scope.activityId = $scope.explorationId;
 
-          $scope.registerShareEvent = function(network) {
-            siteAnalyticsService.registerShareExplorationEvent(network);
-          };
+          $scope.registerShareEvent =
+          siteAnalyticsService.registerShareExplorationEvent;
```

Comment:
Indent by 2 , since this is a continuation line. Optionally, put parentheses around it, e.g.

```
    $scope.registerShareEvent = (
      siteAnalyticsService.registerShareExplorationEvent);
```

Ditto below.